src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,50 @@
-export const PeopleFilters = () => {
+import classNames from 'classnames';
+import { URLSearchParamsInit } from 'react-router-dom';
+import { getSearchWith } from '../utils/searchHelper';
+import { centuriesList } from '../utils/constants';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  searchParams: URLSearchParams;
+  onSetSearchParams: (nextInit: URLSearchParamsInit, navigateOptions?: {
+    replace?: boolean | undefined;
```

Comment:
```suggestion
    replace?: boolean;
```
The question mark implies that it could be `undefined`.